calico_node/filesystem/startup.py

```diff
@@ -261,7 +261,7 @@ def main():
     # This is required because the confd templates expect to be able to fill in
     #  the IP by fetching it from the environment.
     with open('startup.env', 'w') as f:
-        f.write("IP=" + ip)
+        f.write("IP=" + str(ip))
```

Comment:
There was a recent change to specifically have `get_host_ips` return IPAddress objects, but this call was missed during the update.

I'm not opposed to having the other functions use IPAddress objects (it's not a large change). I don't think it will have much of an effect on speed/memory and I don't think we really care that much for this call if it did, this is a one-time run to start the node.